app/user/models.py

```diff
@@ -86,3 +87,12 @@ class User(AbstractBaseUser, PermissionsMixin, TimeStampedModel):
     objects = UserManager()
     REQUIRED_FIELDS = ["username", "password"]
     USERNAME_FIELD = "email"
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    image = CloudinaryField("image")
+    bio = models.TextField(blank=True)
```

Comment:
Have this as a charfield so that you can put a max length, and null true even for cloudinary field so that in case they are not provided it is okay, or you wanted them like that?

Reply:
I've changed to char.
Cloudinary field automatically has the option of null